websites/M/MangaDex/dist/metadata.json

```diff
@@ -25,14 +25,15 @@
 	"service": "MangaDex",
 	"description": {
 		"en": "Read manga online for free at MangaDex with no ads, high quality images and support scanlation groups!",
+		"fr": "Lit des mangas en ligne gratuitement sur MangaDex sans pubs, haute qualité d'images et supporte les groupes de scanlation !",
 		"nl": "Lees manga gratis online op MangaDex zonder advertenties, afbeeldingen van hoge kwaliteit en ondersteunende scanlatiegroepen!"
 	},
 	"url": [
 		"mangadex.org",
 		"www.mangadex.org",
 		"mangadex.cc"
 	],
-	"version": "3.0.0",
+	"version": "3.1.0",
```

Comment:
```suggestion
	"version": "3.0.1",
```

Reply:
Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8